### ISSUE TITLE
FBXLoader messes the animation up

### ISSUE THREAD

**Issue by makc** [mrdoob/three.js#14204]
I dont really know why, but this model here looks like this if animated:
![screen shot 2018-06-03 at 2 02 30](https://user-images.githubusercontent.com/242577/40881564-3d91e5aa-66d2-11e8-84f6-e09993d81bef.png)

It does work perfectly fine in autodesk fbx review. There are warnings of dropped weights, and some vertices appear to have as much as 12 influences according to FBXLoader, however it is hard to believe that is the cause (hello #12127) and also previous revisions of the same model in dae format had no such problem.
[Head_69.fbx.zip](https://github.com/mrdoob/three.js/files/2065468/Head_69.fbx.zip)

**Comment by looeee** [comment]
Me, everytime someone reports and FBXLoader bug: "please don't be a problem with bone matrices, please don't be a problem with bone matrices, ..."

But yeah, this is a probably a problem with the bone matrices. 

It doesn't look as bad as your image when I'm testing it though, have you updated to the latest files?

![recording](https://user-images.githubusercontent.com/5307958/40883474-49c23e82-6728-11e8-8a2a-3c4cf4e552ed.gif)

**Comment by makc** [comment]
> It doesn't look as bad as your image when I'm testing it

that's because you are playing 1st clip. try 2nd or 4th

**Comment by looeee** [comment]
Are you able to edit the vertex weights? I think that the next step is to confirm whether this a weights issue or a bone matrices issue. 

If you can reexport this with 4 weights per vertex and the issues are not resolved then we can look at the bone matrices.

**Comment by makc** [comment]
@looeee I'm affraid too many influences is itself a bogus problem, like I said in previous dae version there was no such situation. I don't have latest version in dae format to prove this. But I can, for example, upload fbx file to playcanvas and [it works there](https://youtu.be/aKrtFlUJ_co). Do you know their bones per vertex limit?

**Comment by looeee** [comment]
No idea. But perhaps they do something fancier than we do when re-weighting vertices. All we currently do is to take the 4 biggest weights and delete the rest. 

However, I agree that it's the less likely cause of this. 

Unfortunately the transforms involved in converting FBX bones to three.js bones are pretty complex - especially since, as I've understood it, FBX format has been kind of hacked together to support both Max and Maya which specify transforms in different ways. Not to mention MotionBuilder, Blender and all the other apps that may be doing things differently again, or exporting incorrectly. 

While working on setting up the bone matrices I mainly concentrated on Max exports, while making sure not to break any older test files. It's mainly worked for other apps too, but I have come across a couple of issues like this. 

If you want to take a look, it would be great to have a second set of eyes on this. Perhaps you will notice something that I missed.

**Comment by donmccurdy** [comment]
Running this model through Facebook's [FBX2GLTF](https://github.com/facebookincubator/FBX2glTF) utility, the animations match FBX Review:

[Head_69_out.zip](https://github.com/mrdoob/three.js/files/2069699/Head_69_out.zip)

https://gltf-viewer.donmccurdy.com/

**Comment by makc** [comment]
@mrdoob I am trying to, yes. However, if gltf converted from fbx plays fine, it means the problem is definitely there in fbx parser.

**Comment by mrdoob** [comment]
> However, if gltf converted from fbx plays fine, it means the problem is definitely there in fbx parser.

Yes, of course. But we may not have bandwidth to fix it so we're suggesting a more robust workflow.

**Comment by looeee** [comment]
Agreed - I will look into this at some point, but honestly it will be weeks before I have time. If you want a fix sooner, you can try looking into it yourself.

**Comment by yazancash** [comment]
similar problem with all Mixamo FBX animations.

**Comment by mrdoob** [comment]
After #20932 seems like the animations `_WN` and `_WS` are now fixed. `_AN` and `_AS` are still broken though.
